src/devices/uFire_ISE/UFireOrp.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.UFire
+{
+    /// <summary>
+    /// Get ORP measuremens from μFire ISE Probe Interface
+    /// </summary>
+    public class UFireOrp : UFireIse
+    {
+        private const float POTENTIAL_REGISTER_ADDRESS = 0x64;
+
+        /// <summary>
+        /// ORP measuremens
+        /// </summary>
+        public ElectricPotential ORP = new ElectricPotential();
```

Comment:
Please refer as well to naming convention, should be ```Orp``` and as ORP is not a well know abbreviation, should be the full name. Same comment for ISE